test/internal/laws/Applicative.js

```diff
@@ -15,21 +15,21 @@ module.exports = function(equals, A) {
     identity: function(v) {
       var lhs = ap(pure(id))(v);
       var rhs = v;
-      return equals(lhs, rhs);
+      return equals(lhs)(rhs);
```

Comment:
So with this changes API of "laws", from taking `equals` in uncurried form to curried form. rest of API is in uncurried form which could be confusing (in case we move it to FL, but i think it's ok for now).